source/ftrack_connect_pipeline_qt/ui/factory/default/progress_widget.py

```diff
@@ -231,13 +231,14 @@ def prepare_add_steps(self):
         )
         self.content_widget.layout().addWidget(self.status_banner)
 
-    def add_step(self, step_type, step_name, version_id=None):
-        id_name = "{}.{}.{}".format(version_id or '-', step_type, step_name)
-        step_button = PhaseButton(step_name, "Not started")
+    def add_step(self, step_type, name, version_id=None, label=None):
```

Comment:
Changing this argument (step_name to name) breaks consistency as other methods are using step_name.

Reply:
I did this change as progress widget both have widgets steps and stages(for finalizers), this module needs a future cleanup together with the factory. Changing back.